composer.json

```diff
@@ -17,14 +17,18 @@
             "homepage": "http://talis.com/"
         }
     ],
+    "suggest": {
+        "mongodb/mongodb": "MongoDB driver library",
+        "resque/php-resque": "Redis backed library for background jobs"
+    },
     "require": {
         "php" : ">=5.5",
-        "semsol/arc2": "v2.2.4",
-        "chrisboulton/php-resque": "dev-master#98fde571db008a8b48e73022599d1d1c07d4a7b5",
-        "monolog/monolog" : "~1.13",
-        "mongodb/mongodb": "1.0.5"
+        "semsol/arc2": "2.2.6",
+        "monolog/monolog" : "~1.13"
     },
     "require-dev": {
+        "resque/php-resque": "v1.3.4",
```

Comment:
I would prefer to keep this in alignment with RL: https://github.com/talis/rl-app/blob/master/composer.json#L67, namely because of the way it introduces failed-queues

Reply:
OK so buckle up for this :roller_coaster:  

The `chrisboulton/php-resque` referenced here in tripod, is the OG PHP resque implementation, which was later moved to the official resque repository, as `resque/php-resque` package. It literally is the same thing, but still maintained (also, the maintained version includes some fixes that we should be interested in, except we couldn't update to them due to PHP version constraint).

However, what RL references in its composer.json, isn't the same `chrisboulton/php-resque` - it's [our fork](https://github.com/talis/php-resque) where we indeed implemented `RedisMultipleQueues` but, because we didn't change the name of the package, it looks like the OG one, except composer prioritises custom repositories, so it pulls our fork in RL.

So we're already in a situation where it's a different php-resque between tripod and RL. And this is precisely my point - tripod is a library, and it shouldn't impose the configuration of another library on the application (unless it's absolutely needed).

In DC we use the official package with [our package](https://github.com/talis/resque-multiple-failure-queues) that provides the custom job failure backend:

https://github.com/talis/dc-app/blob/2baaedac/composer.json#L67-L68
https://github.com/talis/dc-app/blob/2baaedac/conf/queues.inc.php#L56